app/javascript/src/component_editable_collection_item_menu.js

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Assume this needs to be added to the Documentation list and should also have JS test at some point. Maybe just add a ticket to backlog as minimum action. There's already `test/component_activated_menu_test.js` so that can probably be used as springboard for most of what is required.

Reply:
Added a ticket to the JStests backlog